src/styles/main.scss

```diff
@@ -1 +1,64 @@
-// put code here
+@import "./variables.scss";
+@import "./keyframes.scss";
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.circle {
+  position: relative;
+  height: 400px;
+  width: 400px;
```

Comment:
<img width="266" alt="Screenshot 2023-11-20 at 14 32 58" src="https://github.com/mate-academy/layout_stop-watch/assets/135232001/d73386a0-c62a-4d13-97e5-0ae5eabbc689">
Consider implementing this requirement